Source/WTF/wtf/ThreadAssertions.h

```diff
@@ -31,6 +31,27 @@
 namespace WTF {
 
 class ThreadLikeAssertion;
+WTF_EXPORT_PRIVATE bool isMainThread();
+
+struct MainThreadLike {
```

Comment:
We'll likely need a MainRunLoopThreadLike too at some point. We shouldn't be "MainThread" checks in WebKit2 code, and it would be a bug in the UIProcess because of iOS applications using both WK1 and WK2 (and thus having a WebThread in the UIProcess).